index.html

```diff
@@ -732,6 +788,7 @@ <h2>DisplayMediaStreamConstraints</h2>
 dictionary DisplayMediaStreamConstraints {
   (boolean or MediaTrackConstraints) video = true;
   (boolean or MediaTrackConstraints) audio = false;
+  CurrentTabPreferenceEnum currentTabPreference = "none";
```

Comment:
I could be wrong, but this would be the first time the word "Tab" appears in a web API. I know this was already discussed in the issue, but I can't help thinking we're over-selecting here.

At a conceptual level, the user agent learns whether self-capture (hall of mirrors) is part of the application's use case(s) or not. This seems like valuable information to the user agent, at its most abstract.

It seems less relevant how the user agent ends up acting on this information: whether this applies only to tab-capture or whether it also applies to window capture of the browser window the document is in ... those are really good questions user agents should perhaps be left to figure out, which seems OK to me.

I also don't think it matters if user agents fall short of promises on this today, since this is a hint. E.g. I wouldn't expect browsers to ever exclude full-screen capture over this, but a user agent might take `"exclude"` as a clue it's OK to blur away the undesirable hall-of-mirrors effect for instance. This way the user agent doesn't need to worry it's stepping on apps that rely on self-capture. cc @youennf 

So I'd suggest an iteration on @martinthomson's wording:
```suggestion
  CurrentTabPreferenceEnum selfCapture = "no-preference";
```

Reply:
I'm OK with the s/none/no-preference replacement.
I don't want to accept the s/currentTabPreference/selfCapture suggestion, because this only refers to tabs. (Also note how it's CurrentTabPreferenceEnum, btw.)

Reply:
(with chair-hat off)

I recall, we had rough agreement (not consensus which would require a call on the list) [on](https://www.w3.org/2022/03/15-webrtc-minutes.html#t04) being a "hint", which seems to leave this more as an indicator describing application preference, than an explicit control surface to enact certain specific behavior.

Yes I'm bikeshedding, and perhaps suggesting a bit of abstraction here that wasn't discussed before, but I don't read the last meeting as a final word on naming, and would love to hear from others. I think the question of whether to introduce the term "Tab" to the web platform seems orthogonal to discussion over behavior.

Reply:
> I recall us having rough agreement (not c[on](https://www.w3.org/2022/03/15-webrtc-minutes.html#t04)sensus which would require a call on the list) on this being a "hint", which seems to leave this more as an indicator describing application preference, than an explicit control surface to enact certain specific behavior.

This has not changed. This is a hint. It remains a hint. We're discussing what the hint will be called.

> I think the question of whether to introduce the term "Tab" to the web platform seems orthogonal to discussion over behavior.

If you're not convinced "tab" can be a convenient shorthand in this case (note my `note` in the PR btw), we can go with the following alternative, although I consider it inferior to "tab":
```webidl
  SelfCapturePreference selfCaptureBrowser = "no-preference";

  enum SelfCapturePreference {
    "no-preference",
    "include",
    "exclude",
  };
```

This way:
1. We use the [browser](https://w3c.github.io/mediacapture-screen-share/#dfn-browser) that everyone is already forced to know means "tab" in most relevant contexts.
2. We have not blocked the way for `selfCaptureWindow` to be introduced later, if the WG happens to be interested later.

Reply:
Ping @jan-ivar; PTAL.